presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -197,6 +197,7 @@
     public static final String PARTIAL_RESULTS_MAX_EXECUTION_TIME_MULTIPLIER = "partial_results_max_execution_time_multiplier";
     public static final String OFFSET_CLAUSE_ENABLED = "offset_clause_enabled";
     public static final String VERBOSE_EXCEEDED_MEMORY_LIMIT_ERRORS_ENABLED = "verbose_exceeded_memory_limit_errors_enabled";
+    public static final String PARTITION_STITCHING_ENABLED = "partition_stitching_enabled";
```

Comment:
Lets call it MATERIALIZED_VIEW_DATA_CONSISTENCY_ENABLED

Reply:
Renamed the config accordingly.